app/policies/family_policy.rb

```diff
@@ -49,13 +49,14 @@ def can_modify_family?(user_person)
   def can_broker_modify_family?(broker, broker_staff)
     ivl_broker_account = @record.active_broker_agency_account
     return false unless ivl_broker_account.present?
-    if broker.present?
-      return true if ivl_broker_account.benefit_sponsors_broker_agency_profile_id == broker.benefit_sponsors_broker_agency_profile_id
-    elsif broker_staff.present?
+    broker_role_family_match = ivl_broker_account.benefit_sponsors_broker_agency_profile_id == broker.benefit_sponsors_broker_agency_profile_id if broker.present?
+    if broker_staff.present?
+      return true if broker_role_family_match
       staff_account = broker_staff.detect{|staff_role| staff_role.benefit_sponsors_broker_agency_profile_id == ivl_broker_account.benefit_sponsors_broker_agency_profile_id}
       return false unless staff_account
       return true if ivl_broker_account.benefit_sponsors_broker_agency_profile_id == staff_account.benefit_sponsors_broker_agency_profile_id
     end
+    return true if broker_role_family_match
```

Comment:
This seems like redundant check. Can you just do below by removing lines 52, 54 and 59 ?

```
return true if  broker.present? && ivl_broker_account.benefit_sponsors_broker_agency_profile_id == broker.benefit_sponsors_broker_agency_profile_id

if broker_staff.present?
...
end

```
 may be you can extract conditional statement into a method def if line length is too long.